helm/hades/templates/hades-operator-role.yaml

```
@@ -0,0 +1,47 @@
+{{- if .Values.hadesOperator.clusterWide }}
+
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: {{ .Values.rbac.hadesOperator.role.name }}
+rules:
+  - apiGroups: [""]
+    resources: ["configmaps","events","pods","pods/log","secrets"]
```

YAMLlint (1.37.1)

[warning] too few spaces after comma

(commas)